sdk/python/tests/test_evaluator_runtime.py

```
@@ -0,0 +1,847 @@
+        evaluator,
+        WorkerConfig(
+            server_url="https://cloud.example",
+            credential="secret",
+            worker_id="worker-test",
+            max_concurrency=1,
+        ),
+        client=client,
+    )
+    asyncio.run(runtime.process_assignment(client.assignment))
+    assert peak == 1
+    DefinitionsResponse,
```

Ruff (0.16.2)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)